### PR TITLE
#175636833 Integrate Travis with read me badge

### PR DESCRIPTION
**Pull request by roclena**
**What does this PR do?**

- Add Travis Badge on readMe

 **Description of Task to be completed?**

- create ` .travis.yml ` 
- Add a link to ReadMe

**How should this be manually tested?**

- run   ` git clone https://github.com/atlp-rwanda/tech-strikers-bn-backend.git`
- run  `git checkout -b ch-integrate-travis-#1756368833`
- push it on your GitHub 
- visit your Readme file
 
 **Any background context you want to provide?**

no

**What are the relevant pivotal tracker stories?**

- [#175636833](https://www.pivotaltracker.com/n/projects/2475704)

 **Screenshots (if appropriate)**

image

**Questions:**

### CODE REVIEW

**Comment by kipropbrian** [comment]
Hey Roger

You need to rebase this PR.
You also need to update the description. Check how others are writing their description
You also need to get 2 reviews from your teammates/

**Review by Duncanwar — COMMENTED** [review]
Rebase Develop

**Comment by Duncanwar** [comment]
Resolve conflicts and use back ticks for command in PR

**Review by MChristian00 — CHANGES_REQUESTED** [review]
Great work so far. Can you please work on the following:
- Rebase. New changes have been merged to Develop and you need to rebase for the conflicts to be resolved.
- Your commit contains two contributors while it should only be showing you. Please work on it.
- As the other reviewers noted, please work on your PR Description.

Good Luck !!

**Comment by bigelio** [comment]
Good job Roger but  try to make your description more descriptive, You can just remove somethings like this **node_js:
- 12.18.4
- node**
Instead of doing like this **_clone using: git clone https://github.com/atlp-rwanda/tech-strikers-bn-backend.git_** you can do it in this way `run git clone https://github.com/atlp-rwanda/tech-strikers-bn-backend.git`.

Use backticks

**Comment by bigelio** [comment]
Remember to do rebasing also.

**Review by geraudirad — APPROVED** [review]

**Review by Duncanwar — APPROVED** [review]
Now it looks ok

**Comment by bigelio** [comment]
Good job @reclina . It looks good.

**Review by bigelio — APPROVED** [review]

File:  [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# .       create a merge commit using the original merge commit's
+# .       message (or the oneline, if no original merge commit was
+# .       specified). Use -c <commit> to reword the commit message.
+#
+# These lines can be re-ordered; they are executed from top to bottom.
+#
+# If you remove a line here THAT COMMIT WILL BE LOST.
+#
+# However, if you remove everything, the rebase will be aborted.
+#
```
Comment (kipropbrian) [inline ]: Delete this file, its not necessary

**Pull request merged**